### PR TITLE
Update .eslintrc

### PR DESCRIPTION
**Pull request by cibernox**
This simple PR is to verify that failures in CI in ember-canary are not because of the changes in other PRs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,5 +34,6 @@
     "ember-cli-release": "^1.0.0-beta.1",
     "exists-sync": "0.0.3",
     "loader.js": "^4.0.10"
```
Comment (trentmwillis) [inline package.json]: Missing a comma here

File: bower.json [hunk 2]
```diff
@@ -24,13 +24,16 @@
     "tests"
   ],
   "dependencies": {
-    "ember": "^2.7.0"
+    "ember": "components/ember#canary"
```
Comment (trentmwillis) [inline bower.json]: Is this change really needed? The ember-try config already covers canary.

Comment (cibernox) [inline bower.json]: No, it's not. I just wanted to share a branch someone could clone and run to see the error with ember-canary. Something very weird is happening at build/transpile level that `emberBabel.classCallCheck` is undefined. 

I plan to remove this second commit.

**Comment by Turbo87** [comment]
@cibernox is this PR still relevant?

**Comment by cibernox** [comment]
Unsure. I'll check tomorrow.

**Comment by rwjblue** [comment]
Fixed with 0.7.0-beta.1 (migrated to a real addon)
